docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -3,19 +3,39 @@
 include::_attributes.adoc[]
 :categories: security,web
 
-The Quarkus Security framework supports multiple authentication mechanisms, which you can use to secure your applications. 
-Some of the supported authentication mechanisms are built into Quarkus and some require you to add an extension.
-You can also combine authentication mechanisms. 
+The Quarkus Security framework supports multiple authentication mechanisms, which you can use to secure your applications.
+Some supported authentication mechanisms are built into Quarkus, and some require you to add an extension.
+You can also combine authentication mechanisms.
 
 Before you choose an authentication mechanism for securing your Quarkus applications, review the information provided.
 
+.Authentication requirements and mechanisms
+[source, adoc]
+|===
+^|Authentication Requirement ^| Authentication Mechanism
+
+^|Username and password ^|xref:security-basic-authentication-concept.adoc[Basic], xref:security-authentication-mechanisms-concept.adoc#form-auth[Form]
+
+^|Bearer Access Token  ^a| xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer], xref:security-jwt.adoc[JWT], xref:security-oauth2.adoc[OAuth2]
+
+* See the <<table>> table below.
+
+^|Single Sign-On (SSO) ^|xref:security-oidc-code-flow-authentication-concept.adoc[OIDC Web], xref:security-authentication-mechanisms-concept.adoc#form-auth[Form]
+
+^|Client Certificate  ^|xref:security-authentication-mechanisms-concept.adoc#mutual-tls[Mutual TLS (MTLS)]
+
+^|WebAuthn ^|xref:security-webauthn-concept.adoc[WebAuthn]
+
+^|Kerberos ticket    ^|link:https://quarkiverse.github.io/quarkiverse-docs/quarkus-kerberos/dev/index.html[Quarkus - Kerberos]
+|===
+
 == Built-in authentication support
 
 Quarkus Security provides the following built-in authentication mechanisms:
 
 * xref:security-basic-authentication-concept.adoc[Basic authentication]
-* <<Form-based authentication>>
-* <<mutual TLS authentication>> 
+* <<form-auth>>
+* <<mutual-tls>>
```

Comment:
In the rendered preview, there are brackets "[ ]" appearing around the this "mutual-tls" link. Just want to double-check that is rendering correctly from your side.

Reply:
I saw it as well

Reply:
It renders correctly in the latest version of a preview.